src/pingdom-monitor_test.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+// import "testing"
```

Comment:
@waseem-h Why is this commented?

Reply:
We cant run these tests in our pipeline. They require pingdom api key

Reply:
@rasheedamir are we good to go with this?

Reply:
yeah we can't do anything!